### ISSUE TITLE
Docker - Update script to write client env file directly to Client directory

### ISSUE THREAD

**Issue by ajhollid** [bluewave-labs/bluewave-uptime#380]
Vite interpolates env vars at build time, not run time. 

When the uptime script is run we should write to `Client/.env` and not `Docker/client.env`

The way it currently works is the script writes to `Docker/client.env` and then the client docker file copies `Docker/client.env` to its working directory.

This step is not needed as we can have the script write directly to `Client/.env` and it will be copied to the working directory when the Client docker file runs 

`COPY ./Client .`

TLDR

Change 

```
echo "VITE_APP_API_BASE_URL=\"$input_url\"" > ./Docker/client.env
```

to

```
echo "VITE_APP_API_BASE_URL=\"$input_url\"" > ./Client/.env
```

in `uptime.sh` and remove

```
COPY ./Docker/client.env .env
```

from the Client dockerfile as it will be redundant
